Signal/src/views/ContactTableViewCell.m

```diff
@@ -50,11 +52,11 @@ - (void)configureProgrammatically
 {
     const CGFloat kAvatarSize = 40.f;
     _avatarView = [UIImageView new];
-    _avatarView.contentMode = UIViewContentModeScaleToFill;
     _avatarView.image = [UIImage imageNamed:@"empty-group-avatar"];
     // applyRoundedBorderToImageView requires the avatar to have
     // the correct size.
     _avatarView.frame = CGRectMake(0, 0, kAvatarSize, kAvatarSize);
+    _avatarView.contentMode = UIViewContentModeScaleToFill;
```

Comment:
I thought we found it was better to set the content mode *before* the frame to avoid distortion.